### DISCUSSION TITLE
在Mac上编译出现"I can't find file `SimSun'"

### DISCUSSION THREAD

**Discussion by xskxzr** [tuna/thuthesis#620] [Q&A] [answered]
编译还是能成功，只是有以下信息
<img width="861" alt="截屏2021-03-23 下午3 06 07" src="https://user-images.githubusercontent.com/7037227/112107247-a82b8f80-8be9-11eb-9943-198f7eafd0bd.png">
这正常吗？

**Comment by zepinglee** [comment, marked as answer]
正常，模板会检测是否有 SimSun 并自动调用，因为研究生院建议使用 Windows 的字体。
